src/tools/autocompletion/README.md

```diff
@@ -0,0 +1,12 @@
+# Autocompletion
+
+To get autocompletion to run you need to have python3 installed and the kdb-python Module up and running.
+
+## Bash
+
+## How to get it to run
+To get the bash completion to run you need to run `source find_autocompletion_options.sh`. After that in your console you should be able to type `kdb <TAB><TAB>` and have completion options show up.
```

Comment:
Mounting of specification not present. Error handling for root key not present. I currently get:

```
markus@zbook:~/Projekte/Elektra/current/src/tools/autocompletion$ kdb Traceback (most recent call last):
  File "find_autocompletion_options.py", line 190, in <module>
    result = find_auto_completion_options()
  File "find_autocompletion_options.py", line 60, in find_auto_completion_options
    root = root_key.getMeta('root').value
AttributeError: 'NoneType' object has no attribute 'getMeta'
```